### PR TITLE
Conference program added

### PR DESCRIPTION
**Pull request by davidrizo**
The conference program has been added. The early bird registration has been extended

### CODE REVIEW

File: _conferences/2021/program.html [hunk 1]
```diff
@@ -55,5 +135,156 @@ <h4>19 July 2021, 18.30 - 21:00 PM (CEST)</h4>
...
+    <h4 id="oral1">#1. 20 July 2021, 17:45-19:15 (CEST) </h4>
+    <h5>Tools & Applications</h5>
+    <ul>
+        <li>
+            <strong>Codified Spanish music heritage through Verovio: the online platforms Fondo de Música Tradicional IMF-CSIC and Books of Hispanic Polyphony IMF-CSIC</strong>
+            <br>Emilio Ros-Fábregas
+            <br><em>CSIC-Institución Milá y Fontanals of Research in Humanities, Barcelona, Spain</em>
+        </li>
+        <li>
+            <strong>Alleviating the last mile of encoding: The mei-friend package for the Atom text editor</strong>
```
Comment (ahankinson) [inline _conferences/2021/program.html:149]: Spelling mistake in Werner Goebl’s name

File: _conferences/2021/program.html [hunk 2]
```diff
@@ -55,5 +135,156 @@ <h4>19 July 2021, 18.30 - 21:00 PM (CEST)</h4>
...
+            <br><em>University of Music and Performing Arts Vienna, Austria</em>
+        </li>
+        <li>
+            <strong>Building a Comprehensive Sheet Music Library Application</strong>
+            <br>Klaus Rettinghaus and Alexander Pacha
+            <br><em>Enote GmbH, Germany</em>
+        </li>        
+        <li>
+            <strong>Lohengrin TimeMachine: musicological multimedia made with MELD</strong>
+            <br>David Lewis (1), Kevin Page (1), and Laurence Dreyfus, Laurence (2)
```
Comment (ahankinson) [inline _conferences/2021/program.html:160]: Spelling mistake, “Laurence” is repeated

**Comment by musicEnfanthen** [comment]
Thank you @davidrizo , and @ahankinson for catching the typos. As soon as @ahankinson 's observations are included, it looks ready to be merged.

File: _conferences/2021/program.html (hunk 1 above)
Comment (davidrizo) [reply _conferences/2021/program.html:149]: Corrected

File: _conferences/2021/program.html (hunk 2 above)
Comment (davidrizo) [reply _conferences/2021/program.html:160]: Corrected, thanks for spotting it

**Review by musicEnfanthen — APPROVED** [review]

**Comment by musicEnfanthen** [comment]
Thank you @davidrizo !

**Pull request merged**